### DISCUSSION TITLE
React Native/Expo Support

### DISCUSSION THREAD

**Discussion by cenkakay** [antoniolago/react-gauge-component#44] [Q&A]
Does work with React Native/Expo?

**Comment by dakota-kallas** [comment]
I also want to know the same thing. I would love to be able to use this in my React Native / Expo application. There really isn't a comparable component package available for React Native.

**Comment by antoniolago** [comment]
Unfortunately is not possible to do a drop-in conversion to React Native, you're stuck with Webviews for that.
